source/sdk/math/Random.ooc

```diff
@@ -39,6 +40,16 @@ Random: class {
 	randFloat: static func -> Float {
 		return (rand() as Float) / INT_MAX
 	}
+	/**
+	   :return: a pseudo-random float number with a normal distribution
+	 */
+	randn: static func (sigma := 1.0f) -> Float {
```

Comment:
We should rewrite how Random numbers work, but that's a larger project. For now, rename this function to `generateGaussianFloat`. There should also exist a version of the function that takes `my` as an argument.